lively.components/inputs.cp.js

```diff
@@ -92,7 +92,7 @@ const SearchField = component({
   }, {
     type: Label,
     name: 'placeholder icon',
-    autofit: false,
+    autofit: true,
```

Comment:
What did this solve? :thinking:

Reply:
Once I set that, the "erase input" button was aligned properly.